f1tenth_gym/envs/track/utils.py

```diff
@@ -24,7 +24,7 @@ def find_track_dir(track_name: str) -> pathlib.Path:
     FileNotFoundError
         if no map directory matching the track name is found
     """
-    map_dir = pathlib.Path(__file__).parent.parent.parent.parent / "maps"
+    map_dir = pathlib.Path(__file__).parent.parent.parent.parent.parent / "maps"
```

Comment:
why this? it does not point to the right "maps" directory in this way

```suggestion
    map_dir = pathlib.Path(__file__).parent.parent.parent.parent / "maps"
```